docs/content/howtos/simulate-ransomware.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "How to simulate a ransomware attack"
+draft: false
+pre: '<i class="fa fa-lock"></i> '
+tags: ["howtos", "simulate ransomware"]
+---
+
+Follow these steps to successfully simulate a ransomware attack on your
+network.
+
+1. Prepare your environment for a ransomware simulation
+
+    The Infection Monkey will only encrypt files that are allowed by you.
```

Comment:
I think elsewhere we've been using either "Infection Monkey" or "the Monkey" instead of "the Infection Monkey."

Reply:
I've seen "Infection Monkey" and "the Infection Monkey" in the pages we've updated.

Reply:
Last I heard from @triciakickssaas we should be using "Infection Monkey".